EIPS/eip-4635.md

```diff
@@ -0,0 +1,312 @@
+---
+eip: 4635
+title: Semi-fungible token standard
+author: Bo Lin (@linvictor88), Danyang Li (@muren102), Frank Shen (@d22shen)
+discussions-to: https://ethereum-magicians.org/t/eip-4635-semi-fungible-token-standard/7923
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-12-16
+requires: 721
+---
+
+## Simple Summary
+A standard interface for contracts that manage semi-fungible token(s). A single deployed contract usually includes multiple types of fungible tokens and one single transformational non-fungible token.
+
+## Abstract
+This standard outlines a smart contract interface that can represent semi-fungible tokens (SFT). SFTs start out as types of fungible tokens and finally end as non-fungible tokens to provide value. ERC-1155 can manage combination of fungible tokens and non-fungible tokens, however, it's intractable for semi-fungible token representation. So this standard is specially for semi-fungible token use, in which fungible token types are managed like ERC-1155 and non-fungible tokens are totally managed by ERC-721.
```

Comment:
It's not clear to me what the term "semi-fungible" means here?

Reply:
SFTs are a relatively new group of tokens that can be both fungible and non-fungible during their lifecycle. 
e.g. the art museum tickets, it is FT in the beginning. and when it is bought by someone, the ticket will be updated with specific ID and name and now it is NFT.